packages/svelte-urql/src/queryStore.ts

```diff
@@ -151,15 +155,20 @@ export function queryStore<
           return concat<Partial<OperationResultState<Data, Variables>>>([
```

Comment:
Let's invert this here so `fetching: true` applies, i.e. placing concat inside `switchMap`